tests/SemverTest.php

```diff
@@ -72,7 +72,7 @@ public function testUsortShouldInitialVersionParserClass()
         $manipulateVersionStringMethod->setAccessible(true);
         $result = $manipulateVersionStringMethod->invoke(new Semver(), $versions, 1);
 
-        $this->assertInternalType('array', $result);
+        $this->assertIsArray($result);
```

Comment:
This breaks the tests on PHP 5.3 and 5.4.

Reply:
Alright. I'll revert it.
Couldn't run the tests myself anyway. The dependencies which will be installed if I call composer test  aren't compatible with the code on PHP 7.4.

Reply:
To run tests locally on 7.4 you need to run: `export SYMFONY_PHPUNIT_VERSION=7.5; composer test` - I'll need to fix this mess at some point :)

Reply:
@Seldaek I prepared a patch for the tests which works with Symfony 5.2. Roadmap says it will be released in 1-2 months . 
https://github.com/composer/semver/compare/3.2.1...simonberger:limit-phpunit-version

Reply:
@simonberger sounds good if you wanna open a PR with this so it doesn't get lost.